test/intl402/NumberFormat/prototype/format/value-tonumber.js

```diff
@@ -19,7 +19,9 @@ const toNumberResults = [
   [true, 1],
   [false, 0],
   ['42', 42],
-  ['foo', NaN]
+  ['foo', NaN],
+  ['Infinity', 'Infinity'],
+  ['-Infinity', '-Infinity']
```

Comment:
This doesn't test much; the test is

```

toNumberResults.forEach(pair => {
  const [value, result] = pair;
  assert.sameValue(nf.format(value), nf.format(result));
});
```

so you're only testing that `nf.format('Infinity')` is the same value as `nf.format('Infinity')`, which is an identity.

Reply:
Ah, that's right. I've changed the second element in each case to be the Number representation of Infinity.